legal-api/report-templates/template-parts/common/businessDetails.html

```diff
@@ -48,7 +48,11 @@
                     {% if business.legalType in ['GP', 'SP'] and business.state == 'HISTORICAL' %}
                         <span>Dissolved</span>
                     {% else %}
-                        <span>{{stateFilings[0].filingName}}</span>
+                        {% if stateFilings[0].filingName == 'Amalgamated Into' %}
+                           <span>Amalgamated</span>
+                        {% else %}
+                           <span>{{ stateFilings[0].filingName }}</span>
+                        {% endif %}
```

Comment:
No need of `if`. Update it here https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/reports/business_document.py#L583